libraries/AC_AttitudeControl/ControlMonitor.cpp

```diff
@@ -41,14 +42,15 @@ void AC_AttitudeControl::control_monitor_update(void)
  */
 void AC_AttitudeControl::control_monitor_log(void)
 {
-    DataFlash_Class::instance()->Log_Write("CTRL", "TimeUS,RMSRollP,RMSRollD,RMSPitchP,RMSPitchD,RMSYaw", "Qfffff",
-                                           AP_HAL::micros64(),
-                                           (double)sqrtf(_control_monitor.rms_roll_P),
-                                           (double)sqrtf(_control_monitor.rms_roll_D),
-                                           (double)sqrtf(_control_monitor.rms_pitch_P),
-                                           (double)sqrtf(_control_monitor.rms_pitch_D),
-                                           (double)sqrtf(_control_monitor.rms_yaw));
-
+    AP::logger().write("CTRL",
```

Comment:
I like this. But then maybe the class/library should be AP_Logger rather than AP_Logging?